### PR TITLE
Add Text Utilities

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

**Review by Omkar-Ghate — APPROVED** [review]
Looks good to merge.

File: components/textUtils/textUtilsPage.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    margin-bottom: 2rem;
+}
+
+.container__code-preview {
+    width: 80rem;
+    min-height: 22rem;
+    margin-top: 5rem;
+    align-self: center;
+    justify-content: center;
+    overflow: scroll;
```
Comment (zoef-yt) [inline components/textUtils/textUtilsPage.css]: just a suggestion. use overflow:auto this have scrollbar only when needed not always.

**Review by zoef-yt — APPROVED** [review]
Looks good to me. Good to merge.

File: components/textUtils/textUtilsPage.css (hunk 1 above)
Comment (varun13169) [reply components/textUtils/textUtilsPage.css]: Oh nice, changed it to auto.

**Pull request merged**